Gem/Code/Source/Components/NetworkPlayerMovementComponent.cpp

```diff
@@ -94,8 +96,33 @@ namespace MultiplayerSample
         }
     }
 
+    bool NetworkPlayerMovementComponentController::ShouldProcessInput() const
+    {
+        AzFramework::SystemCursorState systemCursorState{AzFramework::SystemCursorState::Unknown};
+        AzFramework::InputSystemCursorRequestBus::EventResult( systemCursorState, AzFramework::InputDeviceMouse::Id,
+            &AzFramework::InputSystemCursorRequests::GetSystemCursorState);
+
+        // only process input when the system cursor isn't unconstrainted and visible a.k.a console mode
```

Comment:
is constrained >? isn't unconstrainted

Reply:
I'm not sure I understand your question, can you restate?